src/commands/createFunction/FunctionSubWizard.ts

```diff
@@ -94,9 +91,8 @@ export class FunctionSubWizard {
                 }
             }
 
-            if ((!template.isHttpTrigger && !template.isSqlBindingTemplate) && !canValidateAzureWebJobStorageOnDebug(context.language) && !await getAzureWebJobsStorage(context, context.projectPath)) {
-                promptSteps.push(new AzureWebJobsStoragePromptStep());
-                executeSteps.push(new AzureWebJobsStorageExecuteStep());
+            if (context.newDurableStorageType) {
```

Comment:
Wait, any idea why this was changed? I don't feel like this is equivalent.

Reply:
See above!

Reply:
Sorry for all the double comments. My internet was very spotty in the car

Reply:
Haha yeah I figured, no worries!